test/e2e/operator.go

```diff
@@ -321,3 +322,88 @@ var _ = XDescribe("ARO Operator - MachineSet Controller", func() {
 		}
 	})
 })
+
+var _ = Describe("ARO Operator - ImageConfig Controller", func() {
+	FSpecify("Operator should add appropriate registries", func() {
+		dockerPodSpec := &corev1.Pod{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "Pod",
+				APIVersion: "v1",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   "docker-alpine-pod",
+				Labels: map[string]string{"imgcfg": "e2e"},
+			},
+			Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name:  "alpine",
+						Image: "alpine",
+					},
+				},
+				RestartPolicy: "Never",
+			},
+		}
+		quayPodSpec := &corev1.Pod{
+			TypeMeta: metav1.TypeMeta{
+				Kind:       "Pod",
+				APIVersion: "v1",
+			},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:   "quay-alpine-pod",
+				Labels: map[string]string{"imgcfg": "e2e"},
+			},
+			Spec: corev1.PodSpec{
+				Containers: []corev1.Container{
+					{
+						Name:  "alpine",
+						Image: "quay.io/kmagdani/alpine",
```

Comment:
The final version will have arosvc.azurecr.io instead of quay.io! Since that is one of the registries that must be allowed.  this is a placeholder until I find an appropriate image in arosvc

Reply:
Furthermore, i want to find an image that is not from a personal registry as you mentioned! more on this will be coming